Cargo.toml

```diff
@@ -16,4 +16,10 @@ keywords = ["data-structures"]
 readme = "README.md"
 
 [features]
-nightly = []
+nightly = ["serde"]
+
+[dependencies]
+serde = { optional = true, version = "^0.7" }
+
+[dev-dependencies]
+serde_json = "^0.7"
```

Comment:
The tests in `tests/serde.rs` don't appear to run with `cargo test`, so how can someone tell if they broke the serde code in that scenario?